trio/_core/_tests/test_guest_mode.py

```diff
@@ -72,7 +72,7 @@ def done_callback(outcome):
             elif op == "unwrap":
                 return obj.unwrap()
             else:  # pragma: no cover
-                assert False
+                raise AssertionError()
```

Comment:
same

Reply:
Changed to `NotImplementedError` with reason in a5afed7